addons/missileguidance/functions/fnc_onFired.sqf

```diff
@@ -126,10 +126,8 @@ private _navigationStateSubclass = _config >> "navigationStates";
 private _states = getArray (_navigationStateSubclass >> "states");
 
 private _navigationStateData = [];
-private _initialState = "";
```

Comment:
I think this might have been planned for line 181, but it looks like using index=0 as default should be fine